### ISSUE TITLE
Feature Map crash when opening a file

### ISSUE THREAD

**Issue by joshwatson** [Vector35/binaryninja-api#1268]
Version: 1.1.1493-dev

I also saw in stderr, `QObject::connect: No such slot QWidget::updateTimerEvent()` but I don't know if the two are related or not. I'm assuming as much though, give the stack trace:

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   org.qt-project.QtCore         	0x0000000106e9dde3 0x106c88000 + 2186723
1   org.qt-project.QtCore         	0x0000000106e9d3c4 QObject::connect(QObject const*, char const*, QObject const*, char const*, Qt::ConnectionType) + 2212
2   com.vector35.binaryninja      	0x0000000103fa07bc FeatureMap::FeatureMap(QWidget*, ViewFrame*, BinaryNinja::Ref<BinaryNinja::BinaryView>) + 2652
3   com.vector35.binaryninja      	0x0000000103f2b7d1 ViewFrame::ViewFrame(QWidget*, FileContext*, QString const&) + 1201
4   com.vector35.binaryninja      	0x0000000103ed0da0 0x103eb8000 + 101792
5   com.vector35.binaryninja      	0x0000000103ed1d26 0x103eb8000 + 105766
6   com.vector35.binaryninja      	0x0000000103ed1761 0x103eb8000 + 104289
7   com.vector35.binaryninja      	0x0000000103f787f6 0x103eb8000 + 788470
8   org.qt-project.QtCore         	0x0000000106ea1d7b QMetaObject::activate(QObject*, int, int, void**) + 2219
9   org.qt-project.QtWidgets      	0x000000010618ca15 QAction::activate(QAction::ActionEvent) + 309
10  org.qt-project.QtCore         	0x0000000106e9a801 QObject::event(QEvent*) + 753
11  org.qt-project.QtWidgets      	0x0000000106195f7d QApplicationPrivate::notify_helper(QObject*, QEvent*) + 269
12  org.qt-project.QtWidgets      	0x0000000106197382 QApplication::notify(QObject*, QEvent*) + 594
13  org.qt-project.QtCore         	0x0000000106e70fb4 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 212
14  org.qt-project.QtCore         	0x0000000106e721ee QCoreApplicationPrivate::sendPostedEvents(QObject*, int, QThreadData*) + 878
15  libqcocoa.dylib               	0x0000000109a11b89 0x1099e4000 + 187273
16  libqcocoa.dylib               	0x0000000109a12400 0x1099e4000 + 189440
17  com.apple.CoreFoundation      	0x00007fff3688c405 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
18  com.apple.CoreFoundation      	0x00007fff3688c3ab __CFRunLoopDoSource0 + 108
19  com.apple.CoreFoundation      	0x00007fff3686fe51 __CFRunLoopDoSources0 + 195
20  com.apple.CoreFoundation      	0x00007fff3686f3fa __CFRunLoopRun + 1219
21  com.apple.CoreFoundation      	0x00007fff3686ece4 CFRunLoopRunSpecific + 463
22  com.apple.HIToolbox           	0x00007fff35b08895 RunCurrentEventLoopInMode + 293
23  com.apple.HIToolbox           	0x00007fff35b085cb ReceiveNextEventCommon + 618
24  com.apple.HIToolbox           	0x00007fff35b08348 _BlockUntilNextEventMatchingListInModeWithFilter + 64
25  com.apple.AppKit              	0x00007fff33dc595b _DPSNextEvent + 997
26  com.apple.AppKit              	0x00007fff33dc46fa -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1362
27  com.apple.AppKit              	0x00007fff33dbe75d -[NSApplication run] + 699
28  libqcocoa.dylib               	0x0000000109a1125b 0x1099e4000 + 184923
29  org.qt-project.QtCore         	0x0000000106e6c61f QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 431
30  org.qt-project.QtCore         	0x0000000106e715c2 QCoreApplication::exec() + 130
31  com.vector35.binaryninja      	0x0000000103ec39ac 0x103eb8000 + 47532
32  libdyld.dylib                 	0x00007fff63909085 start + 1

Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x200007ff0d3a06a3  rbx: 0x00000001071c70d0  rcx: 0x0000000000000000  rdx: 0x00007ff0d3ab67a0
  rdi: 0x00000001071a2c28  rsi: 0x00007ff0ce409790  rbp: 0x00007ffeebd44ec0  rsp: 0x00007ffeebd44e80
   r8: 0x000000000000000f   r9: 0x00007ff0d320adf0  r10: 0x0000000000000010  r11: 0x0000000000000010
  r12: 0x00007ff0d3a4ebb8  r13: 0x0000000000000001  r14: 0x00000001071c70d0  r15: 0x00000001071a2c28
  rip: 0x0000000106e9dde3  rfl: 0x0000000000010206  cr2: 0x0000000106ca54c0
```

**Comment by bpotchik** [comment]
I believe this the same memory problem as #1259. So far it's been difficult to repro. It might be related to the action system, or parenting of a QSplitter.

**Comment by bpotchik** [comment]
Fixed in build 1.1.1496-dev.
